packages/web/src/components/duplicate-add-confirmation-modal/DuplicateAddConfirmationModal.tsx

```diff
@@ -62,7 +66,7 @@ export const DuplicateAddConfirmationModal = () => {
       if (account) {
         toast(
           <ToastLinkContent
-            text={messages.addedToast}
+            text={messages.addedToast(collectionType)}
```

Comment:
great job finding all this areas where we need to update from playlist to collection, really clutch.